### PR TITLE
Fix issue #2

### PR DESCRIPTION
**Pull request by maoschanz**

### CODE REVIEW

File: src/main/java/fr/inria/diverse/pdedependencies2dot/Pdedependencies2model.xtend [hunk 1]
```diff
@@ -174,16 +174,15 @@ class Pdedependencies2model {
...
 
 			val plugin = findPluginOrCreate(name)
 			plugin.processed = true
-
+			
 			if(allRequired !== null && !allRequired.equals("")) {
 				for (r : allRequired.split(",")) {
 					val rname = parseManifestValue(r)
-					if(okPrefix(rname)) {
+					if(okPrefix(rname) && rname.indexOf("\"") == -1) {
```
Comment (ebousse) [inline src/main/java/fr/inria/diverse/pdedependencies2dot/Pdedependencies2model.xtend:193]: That's rather clever! 

However both to make the code more readable and to cover more cases, I would prefer to better parse and eliminate all **parameters** declared after the package name, ie. each thing declared after a semicolumn.

Here is the OSGI specification: https://osgi.org/specification/osgi.core/7.0.0/framework.module.html#framework.module.requirebundle

It shows as an example that this line is possible and valid:
```Require-Bundle: com.acme.bar._nl;visibility:=reexport;resolution:=optional,```

where two parameters are declared, **and neither contains a quote!** therefore your approach would not work.

And this is because the grammar for `Require-Bundle` is as follows:
```
Require-Bundle      ::= bundle-description 
    ( ',' bundle-description )*
bundle-description  ::= symbolic-name       // See 1.3.2
    (';' parameter )*
```

In conclusion: it would be great to properly parse and eliminate the `(';' parameter )*` part of each `Require-Bundle` before doing the split. Then we could be certain that we only have a list of plugin names.

**Review by ebousse — CHANGES_REQUESTED** [review]
Thanks for the PR! I made an inline comment on your proposal, tell me what you think.

File: src/main/java/fr/inria/diverse/pdedependencies2dot/Pdedependencies2model.xtend (hunk 1 above)
Comment (maoschanz) [reply src/main/java/fr/inria/diverse/pdedependencies2dot/Pdedependencies2model.xtend:193]: i thought the semicolon thing was already correctly done by `parseManifestValue` method?

Comment (ebousse) [reply src/main/java/fr/inria/diverse/pdedependencies2dot/Pdedependencies2model.xtend:193]: True! I forgot that this was already taken into account. Forget about my comment then.

Then my only request is the following one: can you add a comment in the code to explain why you check for the presence of a quote char?

**Comment by ebousse** [comment]
Thanks! I squashed your commits, and changed the description of the resulting commit to more clearly refer to the original problem.
